src/descriptorCache.ts

```diff
@@ -1,6 +1,5 @@
 import * as fs from 'fs'
-import { SFCDescriptor } from '@vue/compiler-sfc'
-import { parse } from '@vue/compiler-sfc'
+import { SFCDescriptor, parse } from '@vue/compiler-sfc'
```

Comment:
It's better to change the first statement to `import type` than to merge them. They are different types of imports.